drivers/barometer_bmp085.cpp

```diff
@@ -88,7 +88,7 @@ const uint8_t BMP085_READPRESSURECMD    = 0x34;         ///< Read Pressure Comma
 // PUBLIC FUNCTIONS IMPLEMENTATION
 //------------------------------------------------------------------------------
 
-Bmp085::Bmp085(I2c& i2c):
+Barometer_BMP085::Barometer_BMP085(I2c& i2c):
     i2c_(i2c),
     ac1_(408),
```

Comment:
Actually these variables should be read from the sensor in order to have good calibration.